vm-c/vm.cpp

```diff
@@ -215,6 +215,8 @@ CallResult run_transaction(CallContext ctx) {
   std::string gasLeft = of_z(extracted->gas);
   std::string refund = of_z(extracted->refund);
   std::string status = of_z(extracted->status);
+  std::cerr << len(extracted->statuscode) << std::endl;
+  std::string statusCode = std::string(extracted->statuscode->data, len(extracted->statuscode));
```

Comment:
@dwightguth This throws a segmentation fault. Probably because the length outputted above is `8589935816` ?